test/config/tslint-custom-rules-with-two-dirs.json

```diff
@@ -2,6 +2,7 @@
   "rulesDirectory": ["../files/custom-rules-2", "../files/custom-rules/"],
   "rules": {
     "always-fail": true,
-    "no-fail": true
+    "no-fail": true,
+    "rule-two": true
```

Comment:
what was this for? It seems to break the tests because there is no rule-two in existence for [this test](https://github.com/palantir/tslint/blob/eadbca9389439ab8685ebe0a50fd63a3fe340930/test/check-bin.sh#L63)